runtime/onert/core/src/exec/ExecutorBase.h

```diff
@@ -158,6 +158,11 @@ class ExecutorBase : public IExecutor
   void handleDynamicInputTensor(ir::IOIndex input_index, const IODescription &desc);
 };
 
+/**
+ * @brief Returns @c true if any input tensor is dynamic; @c false if all are static tensors
+ */
+bool has_dynamic_input(const std::vector<std::shared_ptr<onert::backend::ITensor>> &input_tensors);
```

Comment:
All C++ function names must be in `camelCase` 😄

Reply:
Ah. still confused with coding convention in ONE compiler source code. :-) will be fixed.